src/data/pokemon-evolutions.ts

```diff
@@ -373,12 +373,49 @@ export class GholdengoEvoCondition extends SpeciesEvolutionCondition {
   }
 }
 
+/**
+ * Values obtained from the friendship values needed to evolve in generations 2-7 and 8-9
+ */
+const BABY_FRIENDSHIP_EVO_REQUIREMENT = 160;
+const FRIENDSHIP_EVO_REQUIREMENT = 220;
+
+/**
+ * For baby evolutions that require friendship
+ * Used for the following:
+ * Pichu
+ * Cleffa
+ * Igglybuff
+ * Happiny
+ * Togepi
+ * Azurill
+ * Budew
+ * Chingling
```

Comment:
```suggestion
 * For baby evolutions that require friendship.
 *
 * Used for the following:
 * ```
 * Pichu
 * Cleffa
 * Igglybuff
 * Happiny
 * Togepi
 * Azurill
 * Budew
 * Chingling
 * ```
```